modules/openxr/openxr_interface.cpp

```diff
@@ -154,9 +155,14 @@ PackedStringArray OpenXRInterface::get_suggested_tracker_names() const {
 	// These are hardcoded in OpenXR, note that they will only be available if added to our action map
 
 	PackedStringArray arr = {
-		"left_hand", // /user/hand/left is mapped to our defaults
-		"right_hand", // /user/hand/right is mapped to our defaults
-		"/user/treadmill",
+		"head", // XRPositionalTracker for the users head (Mapped f
```

Comment:
Cut off

Reply:
Whups - yes, I was part way through filling it out and then in looking up the technical data I got derailed - fixed.